kwok/charts/crds/karpenter.sh_nodeclaims.yaml

```diff
@@ -361,6 +247,9 @@ spec:
                 - nodeClassRef
                 - requirements
               type: object
+              x-kubernetes-validations:
```

Comment:
Does this automatically work for every field in the spec?

Reply:
Yes, it does

Reply:
Removed from this PR